gettsim/parameters/arbeitsl_geld_2.yaml

```diff
@@ -432,3 +517,7 @@ berechtigte_wohnfläche_eigentum:
   1984-01-01:
     basisgröße: 80
     erweiterung: 20
+  2023-01-01:
+    eigentum: 140
```

Comment:
naming of parameters before 2023 and after 2023 is inconsistent. My suggestion would be to even transform them such that we have the same parameters before and after the reform. (open for alternatives, I hope I haven't made an error converting)

```yaml
  1984-01-01:
    1: 80
    2: 80
    3: 100
    4: 120
    je_weitere_person: 20
  2023-01-01:
    1: 140
    2: 140
    3: 140
    4: 140
    je_weitere_person: 20

Reply:
Also: The description of the parameter needs to be adjusted.